### ISSUE TITLE
No snapshot in notification

### ISSUE THREAD

**Issue by nikunp** [dgreif/ring#1118]
# Bug Report

### Describe the Bug
Getting motion notifications but no snapshot on doorbell or camera

### To Reproduce
 - Start Homebridge
 - Attempt to ...
 - See error

### Expected behavior
 A clear and concise description of what you expected to happen.

### Screenshots/Logs
 If applicable, add screenshots and log output to help explain your problem.

### Additional context
 Add any other context about the problem here.

### Homebridge Ring Config
Post homebridge-ring platform config  _without sensitive information_
```

HIGHLIGHT THIS TEXT AND POST CONFIG HERE
DO NOT INCLUDE YOUR REFRESH TOKEN OR EMAIL/PASSWORD (replace them with xxx)

```

### Environment

 - OS: Rasbian
 - Node.js: v18.12.1
 - NPM: 8.19.2
 - homebridge-ring: 11.6.0
 - homebridge: 4.50.2
 - hoobs: [e.g. 0.4.50 or N/A]

**Comment by MoTechnicalities** [comment]
Confirmed.

Under homebridge-ring v11.7.0 and the new Home architecture available after iOS 16.2, the Ring cameras don't update the snapshots. Only when viewed will a snapshot for that camera update. For reference, A separate non Ring camera regularly updates its snapshot for me.

![image](https://user-images.githubusercontent.com/68468472/208162867-2f83a936-f103-4a30-a3a5-9f157b9a9226.png)

**Comment by Kaiserben** [comment]
How, that's explain why I don't get snapshots too. I was thinking it was due to my new home bridge installation but seems to be a broader issue.

Symptoms: 
- notifications are not showing any snapshot
- not able to get a snapshot with `curl --request POST http://127.0.1.1:xxxx/resource --header "Content-Type:Application/json" --header "authorization: xxx-xxx-xxx" --data '{ "aid": 2, "resource-type": "image" }'  --output "snapshot.jpg"
`

It returns this message in the home bridge logs :
```
17/12/2022 10:18:37] [Ring] No snapshot cached for Portail
[17/12/2022 10:18:37] [Ring] Loading new snapshot into cache for Portail
[17/12/2022 10:18:52] [Ring] Failed to cache snapshot for Portail (15.002s), The camera currently reports that it is online
[17/12/2022 10:18:52] [Ring] Snapshot for Portail (lpd_v2 - Doorbell Pro) failed to refresh after 15 seconds
```
